src/balena-model.ts

```diff
@@ -11,6 +11,13 @@ export type Deferred<T> = Exclude<T, any[]>;
 export type PickDeferred<T, K extends keyof T> = {
 	[P in K]: Deferred<T[P]>;
 };
+export interface WebResource {
```

Comment:
Is this type autogenerated like the rest in this file?

Reply:
Yes :)